instrumentation/servlet/src/main/java/com/splunk/opentelemetry/servlet/ServletInstrumentationModule.java

```diff
@@ -110,4 +122,14 @@ public void set(HttpServletResponse carrier, String key, String value) {
       carrier.addHeader(key, value);
     }
   }
+
+  public static final class RequestCounter {
+    private static final Counter REQUEST_COUNT = Metrics.counter("http.request.count");
```

Comment:
Can we add an example here of adding a tag to the counter? I don't remember how the micrometer APIs work here, but it'll be an important thing to be able to do. maybe just the HTTP verb for starters?

Reply:
Added 👍